tests/runtime/engine/runner.py

```diff
@@ -317,11 +335,23 @@ def run_test(test):
                 print('\tCLEANUP error: %s' % e.stderr)
                 return Runner.FAIL
 
+        def print_befores_and_after_output():
+            if len(befores_output) > 0:
+                print("\tBefore cmd output: ", end="")
+                for out in befores_output:
+                    out = out.encode("unicode_escape").decode("utf-8")
+                    print(out)
+            if after_output is not None:
```

Comment:
nit: Since bpftrace support multiple BEFORE clause, do we need to print the exact command executed here to diffierenciate?

Reply:
Good point. I wouldn't bother with printing the BEFORE command, though. I just fixed the code to show one `Before cmd output: ...` line per BEFORE command and they can be matched to the respective commands by the order.

Feel free to re-review.

Reply:
yeah, this should be sufficient